ReactNativeClient/lib/services/rest/Api.js

```diff
@@ -451,6 +451,23 @@ class Api {
 			return note;
 		}
 
+		if (request.method === 'PUT') {
+			const note = await Note.load(id);
+			const requestBody = JSON.parse(request.body);
+
+			if (!note) throw new ErrorNotFound();
+
+			let updatedNote = Object.assign({}, note, request.bodyJson(this.readonlyProperties('PUT')));
+			updatedNote = await Note.save(updatedNote, { userSideValidation: true });
```

Comment:
You mentioned you borrowed this from the default handler.  So rather than duplicate the code here, why not call the default handler to do this?

Reply:
Yeah, I didn't think about that! I'll implement it now.